TLSharp.Core/TelegramClient.cs

```diff
@@ -15,6 +15,7 @@
 using TLSharp.Core.MTProto.Crypto;
 using TLSharp.Core.Network;
 using TLSharp.Core.Utils;
+using static TLSharp.Core.Network.TcpTransport;

```

Comment:
I would prefer you place the handler outside the class, to avoid this weird `using` line :)